### ISSUE TITLE
Make text size adjustable with iOS accessibility settings

### ISSUE THREAD

**Issue by huertanix** [OpenArchive/Save-app-ios#213]
Per the Apple's HIG guidance (https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/text-size-and-weight/), text sizing should be adjustable via iOS settings, to make it easier for older users and other differently sighted people to use.

**Comment by tladesignz** [comment]
Great catch, thank you!

Situation much improved with upcoming build (63). Please test with next TestFlight.

Note: Some texts might not appear bigger, due to other constraints. E.g. text needs to fit in a line, so font size is scaled down to fit.

I might have overlooked 1 or 2 items. Happy if you find them.

**Comment by huertanix** [comment]
Poked around different parts of the app, text sizing seems to work in most places, but there's some rough edges here and there:

![cancel_done_text_sizing](https://user-images.githubusercontent.com/72914/177054027-82ad7f7e-f50e-4251-9afc-3b25e9d34915.png)

In the media upload dialog, "Cancel" has its size adjusted accordingly, but "Done" does not. Also, the time label for video media stays the same size (this might just be an iOS thing that can't be changed though).

![connect_your_space_text_wrapping](https://user-images.githubusercontent.com/72914/177054308-4b4891b3-eaef-40ed-935e-af6dceb49f4f.png)

In the dialog to connect a new space, some text that gets maybe too big gets lost under the metaphorical fold.

![media_uploads_line_spacing](https://user-images.githubusercontent.com/72914/177054359-b17f50f8-06ff-4a76-a6ce-845bf3e1b04b.png)

In the display for media in a project, the line spacing can use some adjustment according to text size. Also, the project name doesn't get larger with text size settings.

![upload_chunking_small_text](https://user-images.githubusercontent.com/72914/177054500-e2d846e3-53f6-4714-af6b-376cc892cdcf.png)

Upload chunking option label seems to not grow larger with larger text size settings.

**Comment by tladesignz** [comment]
> In the media upload dialog, "Cancel" has its size adjusted accordingly, but "Done" does not. 
> Also, the time label for video media stays the same size (this might just be an iOS thing that can't be changed though).

*sigh* Yet another fork of a dependency: https://github.com/tilltue/TLPhotoPicker/pull/341

> In the dialog to connect a new space, some text that gets maybe too big gets lost under the metaphorical fold.

Fixed.

> Upload chunking option label seems to not grow larger with larger text size settings.

Fixed.


Will try to improve the main scene tomorrow.

Find improvements in upcoming TestFlight build 66.

**Comment by purvi-ranawat** [comment]
@tladesignz  It is working good in most of the places except Select Media screen. Fonts for Recents and Videos do not change.

Tested on version 66.



![Image](https://user-images.githubusercontent.com/8190141/180483889-07a7197f-fa67-4db8-bde1-e6f1738dd9fc.jpeg)

**Comment by tladesignz** [comment]
Good catch, @purvi-ranawat, thank you!

Fix will be contained in next TestFlight (build 67).

**Comment by purvi-ranawat** [comment]
It works fine. Tested build 67.
